test/native/__mocks__/react-native-aztec/index.js

```diff
@@ -1,20 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { Component } from '@wordpress/element';
-
-class AztecView extends Component {
```

Comment:
This mock is not being used because the module path for the `AztecView` component is `@wordpress/react-native-aztec` not `react-native-aztec` so it looks like we don't need it.

In fact I did a test moving this to the path `test/native/__mocks__/@wordpress/react-native-aztec` and when I ran the native unit tests, some snapshots had references to `AztecView` component so I think it's better to not mock it.